gcloud-java-contrib/gcloud-java-nio/src/test/java/com/google/gcloud/storage/contrib/nio/CloudStoragePathTest.java

```diff
@@ -46,75 +46,74 @@
 
   @Rule public final ExpectedException thrown = ExpectedException.none();
 
-  @Before
-  public void before() {
-    CloudStorageFileSystemProvider.setGCloudOptions(LocalGcsHelper.options());
-  }
+  private final FakeStorageRpc storage = new FakeStorageRpc(true);
+  private final NioTestHelper helper = new NioTestHelper(storage);
 
   @Test
-  public void testCreate_neverRemoveExtraSlashes() {
-    try (CloudStorageFileSystem fs = forBucket("doodle")) {
+  public void testCreate_neverRemoveExtraSlashes() throws IOException {
+    try (CloudStorageFileSystem fs = helper.forBucket("doodle")) {
       assertThat(fs.getPath("lol//cat").toString()).isEqualTo("lol//cat");
       assertThat((Object) fs.getPath("lol//cat")).isEqualTo(fs.getPath("lol//cat"));
     }
   }
 
   @Test
-  public void testCreate_preservesTrailingSlash() {
-    try (CloudStorageFileSystem fs = forBucket("doodle")) {
+  public void testCreate_preservesTrailingSlash() throws IOException {
+    try (CloudStorageFileSystem fs = helper.forBucket("doodle")) {
       assertThat(fs.getPath("lol/cat/").toString()).isEqualTo("lol/cat/");
       assertThat((Object) fs.getPath("lol/cat/")).isEqualTo(fs.getPath("lol/cat/"));
     }
   }
 
   @Test
-  public void testGetGcsFilename_empty_notAllowed() {
-    try (CloudStorageFileSystem fs = forBucket("doodle")) {
+  public void testGetGcsFilename_empty_notAllowed() throws IOException {
+    try (CloudStorageFileSystem fs = helper.forBucket("doodle")) {
       thrown.expect(IllegalArgumentException.class);
       fs.getPath("").getBlobId();
     }
   }
 
   @Test
-  public void testGetGcsFilename_stripsPrefixSlash() {
-    try (CloudStorageFileSystem fs = forBucket("doodle")) {
+  public void testGetGcsFilename_stripsPrefixSlash() throws IOException {
+    try (CloudStorageFileSystem fs = helper.forBucket("doodle")) {
       assertThat(fs.getPath("/hi").getBlobId().name()).isEqualTo("hi");
     }
   }
 
   @Test
-  public void testGetGcsFilename_overrideStripPrefixSlash_doesntStripPrefixSlash() {
-    try (CloudStorageFileSystem fs = forBucket("doodle", stripPrefixSlash(false))) {
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Method names should not contain underscores](https://www.codacy.com/app/mziccard/gcloud-java/file/2333823945/issues/source?bid=3123107&fileBranchId=3190440#l84)